### ISSUE TITLE
Node gets renamed without counterpart

### ISSUE THREAD

**Issue by non25** [olrtg/nvim-rename-state#2]
Hi again. :grin: 
I've got latest neovim built from git.
Removed all extra plugins and configuration bits.

<details>
<summary>Plugin list</summary>

```viml
call plug#begin('~/.config/nvim/plugged')

Plug 'neovim/nvim-lspconfig'
Plug 'olrtg/nvim-rename-state'
Plug 'nvim-treesitter/nvim-treesitter', {'do': ':TSUpdate'}

call plug#end()
```

</details>

<details>
<summary>Configuration</summary>

```lua
local lspconfig = require('lspconfig')
local util = require('lspconfig/util')
local on_attach = function(client, bufnr)
  local function buf_set_keymap(...) vim.api.nvim_buf_set_keymap(bufnr, ...) end
  local function buf_set_option(...) vim.api.nvim_buf_set_option(bufnr, ...) end

  require'completion'.on_attach();

  buf_set_option('omnifunc', 'v:lua.vim.lsp.omnifunc')

  local opts = { noremap=true, silent=true }
  buf_set_keymap('n', 'gd', '<Cmd>lua vim.lsp.buf.definition()<CR>', opts)
  buf_set_keymap('n', 'gD', '<cmd>lua vim.lsp.buf.type_definition()<CR>', opts)
  buf_set_keymap('n', 'ga', '<Cmd>lua vim.lsp.buf.code_action()<CR>', opts)
  buf_set_keymap('n', 'K', '<Cmd>lua vim.lsp.buf.hover()<CR>', opts)
  buf_set_keymap('n', 'gr', '<cmd>lua vim.lsp.buf.rename()<CR>', opts)
  buf_set_keymap('n', 'gR', '<cmd>lua vim.lsp.buf.references()<CR>', opts)
  buf_set_keymap('n', 'gc', '<cmd>lua vim.diagnostic.open_float()<CR>', opts)
end

lspconfig.tsserver.setup{
  on_attach = on_attach;
}

require'nvim-rename-state'.setup{}

require'nvim-treesitter.configs'.setup {
  indent = {
    enable = false
  },
  highlight = {
    enable = false
  },
  incremental_selection = {
    enable = false,
    keymaps = {
      init_selection = "gnn",
      node_incremental = "grn",
      scope_incremental = "grc",
      node_decremental = "grm",
    },
  },
}
```
</details>

Would be cool to know how I could debug this further. :grin: 
Thanks.

**Comment by olrtg** [comment]
Hi @non25 ! are you using the command inside a `jsx`/`tsx` file? I was thinking that we could at least print a message if the command couldn't find a counterpart. 🤔

**Comment by non25** [comment]
Hi, updated the plugin and can't reproduce this anymore. :thinking:

**Comment by olrtg** [comment]
Well that's weird! Now I've merged a PR (#5) that allows using the command in `js`/`ts` files just in case. 

Will close this for now then and feel free to open if the issue persist. 😄
